src/components/Slider/Slider.js

```diff
@@ -0,0 +1,75 @@
+import React, {Fragment} from 'react';
+import {Container} from 'react-bootstrap';
+import SliderItem from './SliderItem/SliderItem';
+
+function Slider({title}) {
+  const handleClick = e => {
+    e.preventDefault();
+    const slider = document.querySelector('.slider');
+    const scrollWidth = slider.scrollWidth;
+    e.currentTarget.getAttribute('data-control') === 'prev'
+      ? slider.scrollTo({left: -scrollWidth, behavior: 'smooth'})
+      : slider.scrollTo({left: scrollWidth, behavior: 'smooth'});
```

Comment:
> You can always use custom `data` in html elements.
> Especially in testing it's useful because you can add eg. data-test-name and not worry too much about classList being polluted or using id's :)

fixed